src/operator/tensor/sample_op.h

```diff
@@ -35,13 +36,8 @@ struct SampleUniformParam : public dmlc::Parameter<SampleUniformParam> {
     .describe("Context of output, in format [cpu|gpu|cpu_pinned](n)."
               "Only used for imperative calls.");
     DMLC_DECLARE_FIELD(dtype)
-    .add_enum("None", -1)
```

Comment:
Apparently this was introduced by PR 5019 end of Feb and my changes had been build on top of a version right before that PR. Will merge all of this correctly.